src/panels/calendar/ha-full-calendar.ts

```diff
@@ -60,12 +60,11 @@ const viewButtons: ToggleButton[] = [
 class HAFullCalendar extends LitElement {
   public hass!: HomeAssistant;
 
-  @property({ type: Boolean, reflect: true })
-  public narrow!: boolean;
+  @property({ type: Boolean, reflect: true }) public narrow!: boolean;
```

Comment:
This breaks the card for some reason. When Reflect is set to true on this property and then in the Card, if you set `.narrow=${true}` it freezes the browser... Not sure why this is happening @bramkragten

Reply:
You should be able to just set the attribute `narrow`, is this value always set? Or should you initialize it to false?

Reply:
It should probably be always set by the user of the element right. They set narrow and if not true then false. Should never be a null for a boolean

Reply:
true